mozilla-release/mobile/android/app/src/main/res/layout/cliqz_intro_second_screen_content.xml

```diff
@@ -1,45 +1,46 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<!-- Copyright © Cliqz 2018-->
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:orientation="vertical" android:layout_width="match_parent"
-    android:layout_height="match_parent"
     tools:background="@color/general_blue_color"
-    android:gravity="center_horizontal">
+    tools:gravity="center_horizontal"
```

Comment:
shouldn't you use the android namespace instead of tools?

Reply:
no, I just using tools to display who the view will be, and all this property handled by it's parent